app/api/dapps/management/route.ts

```diff
@@ -44,15 +47,54 @@ export async function GET(req: NextRequest) {
             name: true,
             createdAt: true
           }
+        },
+        emailAlerts: {
+          select: {
+            id: true,
+            email: true,
+            balanceThreshold: true,
+            isActive: true,
+            createdAt: true
+          }
         }
       }
     });
 
-    const formattedDapps = dapps.map((dapp) => ({
-      ...dapp,
-      totalUsed: formattedBalance(dapp.totalUsed),
-      balance: formattedBalance(dapp.balance),
-    }));
+    const formattedDapps = dapps.map((dapp: any) => {
```

Comment:
I think it's could better to be processed on browser side after just returning the data.